kedro-telemetry/tests/test_plugin.py

```diff
@@ -365,11 +363,6 @@ def test_before_command_run_heap_call_error(self, mocker, fake_metadata, caplog)
 
         telemetry_hook.before_command_run(fake_metadata, command_args)
         telemetry_hook.after_command_run()
-        msg = (
```

Comment:
Thanks, @ankatiyar! However, I'm curious—why did we remove the message checks in the tests?

Reply:
They don't show up on the logs if the messages are at debug level

Reply:
Is it possible to change the `caplog` level as done in the previous tests, using
`with caplog.at_level(logging.INFO):`

Reply:
Changed the logging level for tests instead - I was having some trouble with changing it at individual levels. Does this work?

Reply:
Thanks! I don't see any issues with that. I believe it's a good solution!